core/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -230,7 +230,7 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
     ConvertToDeltaCommand(
       visitTableIdentifier(ctx.table),
       Option(ctx.colTypeList).map(colTypeList => StructType(visitColTypeList(colTypeList))),
-      None)
+      ctx.STATISTICS() == null, None)
```

Comment:
Sorry this is not clear, does this returns null when `NO STATISTICS` is present?

Reply:
When `NO STATISTICS` is present, this returns false. Otherwise true.